Formula/pgrouting.rb

```diff
@@ -34,8 +34,13 @@ def install
       system "make", "install", "DESTDIR=#{buildpath}/stage"
     end
 
-    lib.install Dir["stage/**/lib/*"]
-    (share/"postgresql/extension").install Dir["stage/**/share/postgresql/extension/*"]
+    libpq_prefix = Formula["libpq"].prefix.realpath
```

Comment:
Shouldn't it depend on libpq for this?

Reply:
I'm guessing this is due to picking up `libpq` from `postgis -> gdal -> libpq`, which should contains a subset of full `postgresql`.

It would be odd to depend on both `postgresql` and `libpq` (though this is currently happening due to dependency tree).

---

Unrelated side note: `libpq`-dependent formula cause CI linkage warnings when `postgresql` is also install since the RPATH picks up `postgresql` libraries before `libpq` libraries and thinks there is a missing dependency.

Reply:
Hmm that is a weird dependency tree at the moment.  I'm going to see if this can depend only on `libpq`, as I think this would avoid all of the aforementioned issues.

Reply:
It looks like this can just depend on `libpq` without depending on `postgresql` as well.